KenticoListApp/TodoList.Api/App_Start/WebApiConfig.cs

```diff
@@ -15,10 +12,10 @@ public static void Register(HttpConfiguration config)
             config.MapHttpAttributeRoutes();
 
             config.Routes.MapHttpRoute(
-                name: "DefaultApi",
-                routeTemplate: "api/{controller}/{id}",
-                defaults: new { id = RouteParameter.Optional }
+                "DefaultApiV1",
+                "api/v1/{controller}/{id}",

```

Comment:
You should also keep the value for default controller. Now you are exposing the project files and structure on default route...